src/main/java/teammates/common/datatransfer/attributes/EntityAttributes.java

```diff
@@ -57,4 +58,18 @@ public boolean isValid() {
      * e.g. trim strings
      */
     public abstract void sanitizeForSaving();
+
+    /**
+     * Check the list for empty nodes ("") and remove them if found.
+     * Return the list without empty nodes or empty list if all nodes are empty
+     */
+    public List<String> removeEmptyElements(List<String> errors) {
```

Comment:
Thanks, I agree - such naming might be confusing. When I created it I intended to make the name nuetral without mentioning, that it deals with Errors. I though hat maybe in future it might be used to remove empty elements from different lists, but now I see the mistake, you discribed. It was really not clear which type of empty elements the method removes. I changed the name to getNonEmptyErrors()